src/utils/file/index.ts

```diff
@@ -333,7 +333,7 @@ export async function convertForPreview(file: EnteFile, fileBlob: Blob) {
     if (file.metadata.fileType === FILE_TYPE.LIVE_PHOTO) {
         const originalName = fileNameWithoutExtension(file.metadata.title);
         const motionPhoto = await decodeMotionPhoto(fileBlob, originalName);
-        fileBlob = new Blob([motionPhoto.image]);
+        fileBlob = new Blob([motionPhoto.video]);
```

Comment:
so the blob just has the video? when it's paused we wanted to show the motion photo image?

Reply:
The paused state is the start frame of the video which can differ from the image. Will change this.